vl-kontrakt-iaygrunnlag/src/main/java/no/nav/foreldrepenger/kontrakter/iaygrunnlag/arbeid/v1/YrkesaktivitetDto.java

```diff
@@ -133,4 +137,13 @@ public YrkesaktivitetDto medPermisjoner(List<PermisjonDto> permisjoner) {
         this.permisjoner = permisjoner;
         return this;
     }
+
+    public String getNavnArbeidsgiverUtland() {
```

Comment:
Hvis vi kjenner oprinnelses landet så er det bedre om dette modellers med to properties - en landkode og en "arbeidsgiverNavn". Uheldig å kode utland/norge inn i property navn, det er sårbart for utvidelser (1NF)

Reply:
merger nå.  får se på dette etterhvert